definitions/variable/industry/tag_demand_sectors.yaml

```diff
@@ -0,0 +1,27 @@
+- Demand Sector:
+    - Electricity:
+        description: electricity generation sector
+    - Electricity|Biomass:
+          description: electricity generation with biomass power plants
```

Comment:
Would it make sense to change the decsription to "construction of biomass power plants"? Also in the variables below...